src/Sia.Data.Incident/Sia.Data.Incidents.csproj

```diff
@@ -11,17 +12,17 @@
     <Description>Persistence layer for SIA</Description>
     <PackageReleaseNotes>Initial open source release, intended for prototyping.</PackageReleaseNotes>
     <Copyright>Copyright 2017</Copyright>
-    <UserSecretsId>b91df6b9-da48-4846-8071-7a3f379d462d</UserSecretsId>
+    <UserSecretsId>75a727f4-7c4d-4e48-9f77-2d00b67b27e4</UserSecretsId>
```

Comment:
Sia.Data.Incidents now uses the same user secrets file as Sia.Gateway (it was a pain to manage the Sia.Data.Incidents userSecrets file because Visual Studio doesn't give direct access to it).